internal/cli/usercmd/app.go

```diff
@@ -227,6 +227,8 @@ func (c *EpinioClient) AppExport(appName string, directory string) error {
 		return errors.Wrapf(err, "failed to create export directory '%s'", directory)
 	}
 
+	fmt.Println()
```

Comment:
Left over debugging  output ?

Reply:
I don't like it but this was done on purpose to add a newline after the "target directory". If there is a better way to do that I will be happy to remove it. Or I can remove the newline if you prefer.

![image](https://user-images.githubusercontent.com/1763949/199733080-6a5b8a4a-a22c-4dcb-b75a-4da59dff38e4.png)

Reply:
I see. Ok then.